### ISSUE TITLE
"[pylint] E1101:Module 'tf2_ros' has no ... member" even though it exits

### ISSUE THREAD

**Issue by smihael** [PyCQA/pylint#2523]
### Steps to reproduce
1. Open RoboWare Studio 1.2.0 on Ubuntu 16.04 with ROS Kinetic
2. Follow http://wiki.ros.org/tf2/Tutorials/Writing%20a%20tf2%20listener%20%28Python%29

### Current behavior
ExtrapolationException is not found
![screenshot from 2018-09-27 13-15-15](https://user-images.githubusercontent.com/2089122/46142495-77a44e00-c257-11e8-977b-6c6078dcc85c.png)

### Expected behavior
No error since tf2_ros.ExtrapolationException exits.

### pylint --version output

```
pylint --version 
No config file found, using default configuration
pylint 1.5.2, 
astroid 1.4.4
Python 2.7.12 (default, Dec  4 2017, 14:50:18) 
[GCC 5.4.0 20160609]
```

This isn't duplicate of #1975, since there the problem was solved by whitelisting numpy methods.

**Comment by PCManticore** [comment]
@smihael This is an incredibly hard to follow reproduction case. It might be that `tf2_ros` is an extension package, which `pylint` doesn't know to analyze by default. You have a couple of solutions:

- try to see if `--extension-pkg-whitelist=tf2_ros` solves this issue
- try to ignore `tf2_ros` with `--ignored-modules=tf2_ros`
- try to generate that member with `--generated-members=ExtrapolationException`
- finally, check why `pylint` is not able to find that member and write an astroid brain plugin for `tf2_ros` library: https://github.com/PyCQA/astroid/tree/master/astroid/brain
